cluster/gce/util.sh

```diff
@@ -744,10 +744,25 @@ function create-network() {
   fi
 }
 
+function expand-default-subnetwork() {
+  gcloud compute networks switch-mode "${NETWORK}" \
+    --mode custom \
+    --project "${PROJECT}" \
+    --quiet || true
+  gcloud compute networks subnets expand-ip-range "${NETWORK}" \
+    --region="${REGION}" \
+    --project "${PROJECT}" \
+    --prefix-length=19 \
```

Comment:
In the fullness of time yes. But not for 1.7. Now that I think about this the best user experience would be to try using current network (if it exists) and do all this subnet magic if it doesn't.